viewer/packages/camera-manager/src/CameraManagerHelper.ts

```diff
@@ -76,9 +55,16 @@ export class CameraManagerHelper {
    * Updates near and far plane of the camera based on the bounding box.
    * @param camera Used camera instance.
    * @param boundingBox Bounding box of all objects on the scene.
+   * @param nearAndFarPlaneBuffers to read from
+   * @param cameraFarBuffers
    */
-  static updateCameraNearAndFar(camera: PerspectiveCamera, boundingBox: Box3): void {
-    const { cameraPosition, cameraDirection, corners } = this._updateNearAndFarPlaneBuffers;
+  static updateCameraNearAndFar(
+    camera: PerspectiveCamera,
+    boundingBox: Box3,
+    nearAndFarPlaneBuffers: NearAndFarPlaneBuffers,
+    cameraFarBuffers: CameraFarBuffers
+  ): void {
+    const { cameraPosition, cameraDirection, corners } = nearAndFarPlaneBuffers;
```

Comment:
This update breaks the API... :thinking: Is there some other way we can do this?

Reply:
"Getting away" with it doesn't make sense IMHO, since this function would then be a noop since it's writing the results to the input parameters.

Reply:
My suggestion would be to let the function write to the static fields as before if the buffers were not supplied

Reply:
Actually - what if we make a non-static version of this method, with corresponding non-static fields inside CameraManagerHelper? I know it's probably intended to be a static class, but I think it's relatively safe to repurpose it a little. We would still need to keep the static fields/method though, but that would keep the usage simple, without need for extra book-keeping by every camera manager using it